### ISSUE TITLE
distance2point实现有误？

### ISSUE THREAD

**Issue by jstacer1** [talebolano/Contourformer#10]
def distance2point(points, distance, reg_scale):
    
    w = (torch.max(points[...,1]) - torch.min(points[...,1])).clip(min=0)#.detach()
    h = (torch.max(points[...,1]) - torch.min(points[...,1])).clip(min=0)#.detach()

    reg_scale = abs(reg_scale)
    x = points[..., 0] + (0.5 * reg_scale + distance[..., 0]) * (w / reg_scale)
    y = points[..., 1] + (0.5 * reg_scale + distance[..., 1]) * (h / reg_scale)

    new_points = torch.stack([x,y], -1) #

    return new_points
应分别用 x坐标和 y坐标计算宽度和高度吧？

**Comment by talebolano** [comment]
感谢你指出这个错误，我们已经修复并重新上传了权重
